wsl/how-to/install-ubuntu-wsl2.md

```diff
@@ -36,37 +37,38 @@ Ubuntu will then be installed on your machine. Once installed, you can either la
 
 ### Method 2: WSL Command line interface
 
-It is possible to install the same Ubuntu applications available on the Windows Store directly from the command line.
-In a PowerShell terminal, you can run `wsl --list --online` to see all available distros.
+It is possible to install the same Ubuntu versions available on the Windows Store directly from the command line.
+In a PowerShell terminal, you can run `wsl --list --online` to see all available distros and versions.
 
 ![image|690x388](https://github.com/ubuntu/WSL/blob/main/docs/guides/assets/install-ubuntu-wsl2/wsl-list-online.png?raw=true) 
```

Comment:
I'd leave the Windows GUI ones as is (Store UI has changed), the last 3 (Terminal) could use an update to match the 24.04 LTS version in the text, but I'm not sure if you want to keep updating them, say that they show another version (I took the first two and realized I'm only going to have one distribution running - instead of 2 Ubuntus + 2 dockers - for the final screenshot), or remove them (no screenshots for Method 3... and Configure Ubuntu sections)

Reply:
Only update what you can and don't spend too much time on this.

My own preference would be for us to eventually replace some of the terminal images with code blocks, if only for maintenance reasons.

We don't need to solve all of these issues in this PR, however.

Reply:
No problem, in general, I'm usually able to take 1-2 hrs per week for ODA-related edits. 

I've updated the PR with two commits. The [first one](https://github.com/canonical/open-documentation-academy/commit/905ac1f9e4e38577e8618f20f364c54ca18b3074) just adds the changes from your comments, the [second ](https://github.com/canonical/open-documentation-academy/commit/ead326f85c24d519bfe6b07238ba06df9fac076c) replaces the terminal screenshots with text blocks. 

One potential issue that may require more editing is that the how-to progress may be different (including a reboot) if this is the first installation of WSL/ first distro.

Reply:
Thanks - I will try to get to this soon.
At a glance, your changes look solid.

Could you expand on your third point about progress in the how-to and the reboot being required?
I just want to be clear I understand you.

Reply:
On [L19](https://github.com/canonical/open-documentation-academy/blob/76ac672760c17ec0937c2eb1a7538654ffaec068/wsl/how-to/install-ubuntu-wsl2.md?plain=1#L19) there's the single-command first-time installation (with reboot). 
The three methods described later (especially obvious in method 2) refer to a situation when Ubuntu is added to an already existing WSL installation but we're not saying it explicitly. 
A quick workaround for this would could be a change on L21:
from:

Reply:
Only seeing this now (post-review) but will keep it in mind. Thanks.